R/style.R

```diff
@@ -1,5 +1,5 @@
 bullet <- function(lines, bullet) {
-  lines <- paste0(bullet, " ", lines)
+  lines <- paste(bullet, lines)
```

Comment:
Please revert this one. I think we're doing it this way for a reason. It's more explicit.